rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -898,6 +898,30 @@ void getActiveFederationCreationBlockHeight_afterRSKIP186AndNoValueInStorage_ret
 
     }
 
+    @Test
+    void setActiveFederationCreationBlockHeight() {
+
+        // Arrange
+
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP186)).thenReturn(true);
+
+        StorageAccessor storageAccessor = new InMemoryStorage();
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+
+        // Act
+        long expectedValue = 3;
```

Comment:
Same comment as in previous PR, it would nice to use realistic values

Reply:
Updated.